docs/docs/zh/guide/resource/configuration.md

```diff
@@ -2,6 +2,16 @@
 
 资源中心通常用于上传文件、 UDF 函数，以及任务组管理等操作。针对单机环境可以选择本地文件目录作为上传文件夹（此操作不需要部署 Hadoop）。当然也可以选择上传到 Hadoop or MinIO 集群上，此时则需要有 Hadoop（2.6+）或者 MinIOn 等相关环境。
 
+## 本地资源配置
+在单机环境下，可以选择使用本地文件目录作为上传文件夹（无需部署Hadoop），此时需要进行如下配置：
+
+### 配置 common.properties 文件
+
+对以下路径的文件进行配置：`api-server/conf/common.properties` 和 `worker-server/conf/common.properties`
+
+- 将`data.basedir.path`改为本地存储路径，请确保路径存在，并且部署DolphinScheduler的用户拥有读写权限，例如：`data.basedir.path=/tmp/dolphinscheduler`
```

Comment:
```suggestion
- 将 `data.basedir.path` 改为本地存储路径，请确保路径存在，并且部署 DolphinScheduler 的用户拥有读写权限，例如：`data.basedir.path=/tmp/dolphinscheduler`
```

ref :[document specification](https://dolphinscheduler.apache.org/en-us/community/development/document.html). 
Add a space between Chinese and English, for the purpose of document specification.